config/environments/test.rb

```diff
@@ -40,9 +40,9 @@
   config.paperclip_defaults = {
     :storage => :s3,
     :s3_credentials => {
-      :bucket => ENV['S3_BUCKET_NAME'],
-      :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-      :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+      :bucket => ENV['S3_TEST_BUCKET_NAME'],
+      :access_key_id => ENV['AWS_TEST_ACCESS_KEY_ID'],
+      :secret_access_key => ENV['AWS_TEST_SECRET_ACCESS_KEY']
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.